### ISSUE TITLE
Ammy extension couldn't find packages directory.

### ISSUE THREAD

**Issue by pancomputationalist** [AmmyUI/AmmyUI#67]
> "Ammy extension couldn't find packages directory.
> Run `install-package Ammy` in Package Manager Console and restart Visual Studio"

Fresh installation of VS2017 with Xamarin support.
Installed Ammy Extension from [Marketplace](https://marketplace.visualstudio.com/items?itemName=ionoy.Ammy).
Installed `Ammy` and `Ammy.XamarinForms` Nuget packages.
My project directory contains a `packages\Ammy.1.2.38\` folder, but the above error message persists across restarts.

**Comment by ionoy** [comment]
Please try following:

1. Close all files
2. Unload solution
3. Reload solution
4. Open Ammy file

**Comment by pancomputationalist** [comment]
Unloading/Reloading did not help.
However, I noticed that the problem vanishes when I copied the .ammy file from the [Shared project](https://developer.xamarin.com/guides/cross-platform/application_fundamentals/shared_projects/) into the Xamarin.Android project. Now, opening neither file will trigger the error, however, I only get syntax highlighting for the file located in the Android project.
So it seems the extension has problems with handling files in Shared Projects.
